bndtools.core/_plugin.xml

```diff
@@ -370,6 +376,12 @@
             schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
             sequence="M1+M2+F"
       />
+      
+	<key sequence="Ctrl+7"
```

Comment:
Can we somehow use the platform standard for toggle comment?  i think its ctrl+/ or something?